src/ExpressionParser.php

```diff
@@ -48,7 +48,9 @@ class ExpressionParser
 
     private $parser;
     private $env;
+    /** @var array<string, array{precedence: int, class: class-string}> */
     private $unaryOperators;
+    /** @var array<string, array{precedence: int, class: class-string, associativity: int}> */
```

Comment:
`associativity` could even be more precise as `ExpressionParser::OPERATOR_*`

Reply:
I changed to ExpressionParser::OPERATOR_* then